### PR TITLE
Update datafiller data

### PR DESCRIPTION
**Pull request by revolunet**

### CODE REVIEW

File: packages/code-du-travail-data/dataset/datafiller/fetch-requetes.js [hunk 1]
```diff
@@ -29,10 +29,14 @@ const RECORDS_URL = `${DATAFILLER_URL}/kinto/v1/buckets/datasets/collections/req
...
-  if (matches) {
-    return decodeHTML(matches[1]);
+  try {
+    const text = await fetch(url).then(r => r.text());
+    const matches = text.match(/<title>([^<]+)<\/title>/i);
+    if (matches) {
+      return decodeHTML(matches[1]);
+    }
+  } catch (e) {
+    //console.log(e);
```
Comment (lionelB) [inline packages/code-du-travail-data/dataset/datafiller/fetch-requetes.js]: console.error() will not pollute output thus, allow to know wich url fails
```suggestion
    console.error(`fail to retrieve title on ${url}`);
    console.error(e);
```

**Review by lionelB — COMMENTED** [review]
sounds good to me, je fais la modif ?

**Review by lionelB — CHANGES_REQUESTED** [review]
the last item is empty but link to a page 
http://0fec4cb.code-du-travail-numerique.dev.factory.social.gouv.fr/themes/112-formalites-dembauche

**Comment by revolunet** [comment]
closed in favor of #1654

**Pull request closed without merging**